tests/lazy/test_read.py

```diff
@@ -182,3 +185,29 @@ def test_unconsolidated(tmp_path: Path, mtx_format):
     remote_to_memory = remote.to_memory()
     assert_equal(remote_to_memory, adata)
     store.assert_access_count("obs/.zgroup", 1)
+
+
+@pytest.fixture(scope="session")
+def df_group(tmp_path_factory) -> zarr.Group:
+    df = gen_typed_df(120)
+    path = tmp_path_factory.mktemp("foo.zarr")
+    g = zarr.open_group(path, mode="w", zarr_format=2)
+    write_elem(g, "foo", df, dataset_kwargs={"chunks": 25})
+    return zarr.open(path, mode="r")["foo"]
+
+
+@pytest.mark.parametrize(
+    ("chunks", "expected_chunks"),
+    [((1,), (1,)), ((-1,), (120,)), (None, (25,))],
+    ids=["small", "minus_one", "none"],
```

Comment:
I like the explicit IDs you came up with for the other test module!

```suggestion
    ids=["small", "minus_one_uses_full", "none_uses_ondisk_chunking"],
```